sway-ir/src/instruction.rs

```diff
@@ -36,16 +35,16 @@ pub enum Instruction {
     /// Cast the type of a value without changing its actual content.
     BitCast(Value, Type),
     /// An unconditional jump.
-    Branch(Block),
+    Branch((Block, Vec<Value>)),
```

Comment:
I'm looking at where we use these branch variants and seeing `block.1.iter()...` or `block_to_label(&block.0)` and I think it's a little confusing.  Instead of a pair I think it should be a struct, so we can use `branch.block` and `branch.args`.

Reply:
> I'm looking at where we use these branch variants and seeing `block.1.iter()...` or `block_to_label(&block.0)` and I think it's a little confusing. Instead of a pair I think it should be a struct, so we can use `branch.block` and `branch.args`.

Fixed this now.